packages/core/src/components/widgets/SelectWidget.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ChangeEvent, FocusEvent } from "react";
+import { processSelectValue, WidgetProps } from "@rjsf/utils";
+
+function getValue(
+  event: React.SyntheticEvent<HTMLSelectElement>,
+  multiple: boolean
+) {
+  if (multiple) {
+    return Array.from((event.target as HTMLSelectElement).options)
+      .slice()
+      .filter((o) => o.selected)
+      .map((o) => o.value);
+  } else {
+    return (event.target as HTMLSelectElement).value;
+  }
```

Comment:
Because the `return` in the `if() {...}` short-circuits you can actually remove the else...
```suggestion
  }
  return (event.target as HTMLSelectElement).value;
```